assets/js/components/NewChecksResults/ExecutionHeader.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
This component is something we use in other places, right?
Maybe making the filter optional we could have it as reusable component.
Adding the option to change the `span` text and changing the name to something more generic.
Something to have in our radar.

Reply:
Not yet used in other places, however it could be enhanced to make it reusable in the cluster details where we have a similar visualization IIRC.